sites/next.skeleton.dev/src/content/docs/get-started/migrate-from-v2.mdx

```diff
@@ -51,8 +51,22 @@ Please make sure you have accounted for the following:
 
 Skeleton is built on top of the following technologies. These must be migrated individually before proceeding with the Skeleton-specific migration. Note that Svelte and Tailwind provide dedicated CLIs to automate this process.
 
+### Svelte v5
+
 - Migrate to the latest release of Svelte v5 ([migration guide](https://svelte.dev/docs/svelte/v5-migration-guide))
```

Comment:
Removed the `-` bullet points for all three sections. They should just read as paragraphs now.